kong/db/strategies/off/init.lua

```diff
@@ -30,21 +35,16 @@ local _mt = {}
 _mt.__index = _mt
 
 
-local function ws(schema, options)
-  if not schema.workspaceable then
-    return ""
-  end
+local UNINIT_WORKSPACE_ID = "00000000-0000-0000-0000-000000000000"
 
-  if options then
-    if options.workspace == null then
-      return "*"
-    end
-    if options.workspace then
-      return options.workspace
-    end
+
+local function get_default_workspace()
+  if kong.default_workspace == UNINIT_WORKSPACE_ID then
+    local res = kong.db.workspaces:select_by_name("default")
+    kong.default_workspace = assert(res and res.id)
```

Comment:
Should we do this more gracefully?

Reply:
Currently we don't have idea about it, sorry.